src/applications/personalization/dashboard/components/RenderClaimsWidgetDowntimeNotification.jsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { format } from 'date-fns';
 import externalServiceStatus from '~/platform/monitoring/DowntimeNotification/config/externalServiceStatus';
 import DashboardWidgetWrapper from './DashboardWidgetWrapper';
 
-export const RenderClaimsWidgetDowntimeNotification = (downtime, children) => {
-  if (downtime.status === externalServiceStatus.down) {
```

Comment:
How did this even work in the first place 😬 since `children` is a member of the props arg and not a second argument... oof

Reply:
So after taking a look at this and that the Dashboard unit test started failing, I found that the DowntimeNotification component does actually take a prop called `render` which should receive a function with the properties and children as separate args.

This is not the React way, and I'm not exactly sure why it was done this way, but I'm going to change it back so that tests pass again. 😦